pkgs/top-level/python-packages.nix

```diff
@@ -9203,6 +9203,25 @@ let
     };
   };
 
+  mpv = buildPythonPackage rec {
+    name = "mpv-0.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/mpv/${name}.tar.gz";
+      md5 = "48114a00be2dc8898a88b3a253362fae";
+    };
```

Comment:
pypi does md5. As do most packages in this file. Also, what’s wrong with them in this case?